code/modules/projectiles/magazines/flamer.dm

```diff
@@ -249,3 +249,24 @@
 	matter = list("metal" = 3750)
 	flamer_chem = null
 	custom = TRUE
+
+//tanks printable by the research biomass machine
+/obj/item/ammo_magazine/flamer_tank/custom/upgraded
+	name = "Upgraded custom incinerator tank"
+	desc = "A fuel tank used to store fuel for use in the M240 incinerator unit. This one has been modified with a larger and more sophisticated internal propellant tank, allowing for bigger capacity and stronger fuels."
+	matter = list("metal" = 50) // no free metal
+	flamer_chem = null
+	max_rounds = 200
+	max_range = 7
+	fuel_pressure = 1
+	max_duration = 100 //if you manage to reach these caps I will be impressed
+	max_intensity = 100
+	custom = TRUE
+
+/obj/item/ammo_magazine/flamer_tank/smoke/upgraded
+	name = "Large custom incinerator smoke tank"
+	desc = "A tank holding powdered smoke that expands when exposed to an open flame and carries any chemicals along with it. This one has been outfitted with an upgraded internal compressor, allowing for bigger capacity."
+	matter = list("metal" = 50) //no free metal
+	flamer_chem = null
+	custom = TRUE
+	max_rounds = 200
```

Comment:
```suggestion
	max_rounds = 150
```
I'm almost more inclined to just make `/obj/item/ammo_magazine/flamer_tank/smoke` have max rounds at 50 and this be then 100.